minecraft_servers/lostoutlaws/manifest.json

```diff
@@ -1,13 +1,23 @@
 {
   "server_name": "lostoutlaws",
   "nice_name": "LostOutlaws",
-  "direct_ip": "minecraft.lostoutlaws.de",
+  "direct_ip": "lostoutlaws.de",
   "server_wildcards": [
     "%.lostoutlaws.de"
   ],
   "social": {
+	"web_support": "https://discord.lostoutlaws.de",

```

Comment:
Web support means something like a faq or a support web page, not a discord link. Please remove the line.

Reply:
Ah ok then I misunderstood that.
I'll change it as fast as I'm home again! ^^